packages/core/src/components/highlight-box/highlight-box.scss

```diff
@@ -4,6 +4,7 @@
 
 :host {
   display: block;
+  container-type: inline-size;
```

Comment:
Alleen het container type in de `@mixin root() {}` zetten maakte dat in de HTML/CSS implementatie de container queries wel werden gerespecteerd maar niet in het web component.

Reply:
Prima. In dat geval zie ik graag dat er geen `container-type: inline-size` in de `@mixin root()` staat maar direct in de HTML/CSS implementatie.

Nu wordt er 3x `container-type: inline-size` gezet:
* Core `:host` in `packages\core\src\components\highlight-box\highlight-box.scss`
* Core `.dso-highlight-box`, via `@include highlight-box.root()` in `packages\core\src\components\highlight-box\highlight-box.scss`
* HTML/CSS `.dso-highlight-box`, ook via `@include highlight-box.root()` in `packages\dso-toolkit\src\components\highlight-box\highlight-box.scss`.

Uitgangspunt is de styling te centraliseren in mixins, tenzij dat niet kan.